cli/ts/utils.ts

```diff
@@ -142,6 +142,21 @@ const isPathExist = (paths: Array<string>): [boolean, string] => {
     return [true, null]
 }
 
+const CompareOnchainValue = (
+    name: string,
+    onChainValue: any,
```

Comment:
Another naming convention nitpick - here you capitalize the "C" in `onChain` but you didn't in the function name. I suggest we be consistent (i.e. use capital "C" for both)

Reply:
Having seen `onchain` used more commonly, I've decided to go with that. Thanks for the input!